guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -7,8 +7,54 @@ This procedure is only for Katello plug-in and {RHEL}-based operating system use
 endif::[]
 
 Use this procedure to enable the repositories that are required to install {ProductName}.
+Select the option that corresponds with operating system and version you want to install on:
 
-.Procedure
+ifdef::foreman-el,katello,satellite[]
+* xref:#repositories-rhel-8[{RHEL} 8]
+* xref:#repositories-rhel-7[{RHEL} 7]
+endif::[]
+
+ifdef::foreman-el,katello,satellite[]
+== [[repositories-rhel-8]]{RHEL} 8
+
+:distribution-major-version: 8
+:package-manager: dnf
+
+. Disable all repositories:
++
+[options="nowrap"]
+----
+# subscription-manager repos --disable "*"
+----
++
+endif::[]
+
+. Enable the following repositories:
+ifdef::satellite[]
```

Comment:
It won't, change it here and now, I'll get to the review next week.

Reply:
> It won't, change it here and now, I'll get to the review next week.

@maximiliankolb  @melcorr  I don't have repos for non-satellite EL8. Can anyone please share it with me. Evgeni is out of office till 7 June. Need to close this PR at the earliest.

Reply:
@ehelms any chance you could help with this?

Reply:
Okay, in that case it might be a good idea to split it.

Reply:
Thank you @Lennonka 

@maximiliankolb @melcorr Can we proceed further if there are no further reviews?

Reply:
I would have loved to see this resolved with this PR. But we can still merge as is if you create an issue on Github so you/we don't forget about this.

I made one small suggestion and also closed all your resolved threads as part of my re-re-review. Tip: Please comment on open threads in the "review mode" on the code tab and then close all resolved threads so that future reviewers don't have to re-read all your earlier stages.

Please link the issue & I will merge.

Reply:
> made one small suggestion and also closed all your resolved threads as part of my re-re-review.

I have staged it and ready to push.

> But we can still merge as is if you create an issue on Github so you/we don't forget about this.

I will raise the issue here on Github post this PR. As soon as I get the repos from Evgeni or Eric, I will process the same.

> Tip: Please comment on open threads in the "review mode" on the code tab and then close all resolved threads so that future reviewers don't have to re-read all your earlier stages.

Once it is pushed, I am commenting on the review and mark it as Resolved in the 'Conversation' tab. The same reflects on the 'Files Changed' tab. I hope I understand your tip in the correct manner.